lib/src/utilities.dart

```diff
@@ -103,6 +131,9 @@ class SuperTokensUtils {
 
   static Future<LocalSessionState> getLocalSessionState() async {
     var lastAccessTokenUpdate = await getFromStorage(lastAccessTokenStorageKey);
+    // ! DO NOT REMOVE THE BELOW FOR EACH LOOP, REMOVING THIS BREAKS THRE TESTS
+    var instance = await SharedPreferences.getInstance();
+    instance.getKeys().forEach((element) {});
```

Comment:
Is this still relevant? Try running tests without this because this is a very bad thing to add